march_hardware/src/IMotionCube.cpp

```diff
@@ -364,6 +380,84 @@ uint32_t IMotionCube::computeSWCheckSum(int& start_address, int& end_address)
                                  delimiter);
 }
 
+int IMotionCube::verifySetup()
+{
```

Comment:
Maybe put a reference to documentation explaining how this works?

Reply:
Do you not think the docstring in the .h file is sufficient?

Reply:
Hmm that docstring sufficiently explains what the method does, but not how it does that. You write a lot of registers and values, so I think a link to IMC documentation explaining how this works (i.e. what value you need to write to which register) would make it easier for subsequent generations to understand and make alterations where necessary. Does that make sense?

Reply:
I'll provide a reference to the CoE manual and the appropriate chapter and paragraph wherein the process is described in detail.